include/boost/geometry/algorithms/detail/is_valid/polygon.hpp

```diff
@@ -453,7 +476,7 @@ class is_valid_polygon
         // compute turns and check if all are acceptable
         debug_phase::apply(3);
 
-        typedef has_valid_self_turns<Polygon> has_valid_turns;
+        typedef has_valid_self_turns<Polygon, typename Strategy::cs_tag> has_valid_turns;
```

Comment:
why not `typedef`ed as above or not `typedef` the above to look alike. Similar cases below. Am I missing something?

Reply:
No, there is no specific cause. I think I prefer removing `typedef` above because it's used only in one place.

Reply:
Thanks! It is huge, and I could not check all details, but what I've seen looks to me. So I'm OK with merging this.

Reply:
Thanks, I think I'll also add some tests for geometries with undefined CS before merging.